src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.person.Person;
+import seedu.address.storage.Importer;
+import seedu.address.storage.JsonImporter;
+import seedu.address.storage.exceptions.JsonNotFoundException;
+
+public class ImportCommand extends Command {
+    public static final String COMMAND_WORD = "import";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Import the person data contained in an "
+            + "export file. If the person already exists, the personal information and events are updated. "
```

Comment:
Maybe can include that groups and tags are not updated incase some people may get confused!

Reply:
Thanks for the comment! Added this in the latest commit!